[fix] proposal form

rueshyna opened the pull request:
1. fix default value of lambda form
2. modify some terms
3. modify style

for the change 1 and 3, please see the snapshots:
before:
![image](https://user-images.githubusercontent.com/115862/225221561-0f7379c6-e87b-47ca-90eb-f3caf183be06.png)
after:
![image](https://user-images.githubusercontent.com/115862/225221406-a7a84dee-83d9-4447-acfd-7401c5940699.png)

JulesGuesnon reviewed (commented):
Overall LGTM. I think it would be a good idea to remove all the borders in this PR as we removed them everywhere except here I think. Also adding `rounded` class to buttons and inputs so it's more consistent with the rest

rueshyna commented:
> I think it would be a good idea to remove all the borders in this PR as we removed them everywhere except here I think. 

I removed the outer border. This form is in nest structuring so it would be unreadable if we remove all the inner border. We need a redesign for this form.
![image](https://user-images.githubusercontent.com/115862/225594629-cbbbe171-1b3e-4eef-a67a-f1990d6d4c4b.png)


> Also adding rounded class to buttons and inputs so it's more consistent with the rest

In this page, the other form isn't rounded so I still keep it and do a minimal change here. If I have a time, I'll try to do it in next PR :smiling_face_with_tear:.

JulesGuesnon reviewed (approved):
Ok let's go like that for now
